packages/core/src/agent/MessageSender.ts

```diff
@@ -345,6 +337,9 @@ export class MessageSender {
         recipientKeys: queueService.recipientKeys,
         routingKeys: queueService.routingKeys,
         senderKey: firstOurAuthenticationKey,
+        recipientDidDoc: outboundMessageContext.recipientDidDoc,
+        senderDidDoc: outboundMessageContext.senderDidDoc,
+        service: queueService,
```

Comment:
If we pass the service, there's no need to pass the routingKeys and recipientKeys.

Also we now pass the recipientDidDoc and the senderDidDoc. I think we should try to prevent allowing a ton of conflicting options, as it could lead to unexpected behaviour (what If I provide diferent `recipientKeys` and the `service.recipientKeys`